packages/subgraph/abis/Unirep.json

```diff
@@ -0,0 +1,1817 @@
+                "name": "_reputationVerifier",
+                "type": "address"
+            },
+            {
+                "internalType": "contract IVerifier",
+                "name": "_userSignUpVerifier",
+                "type": "address"
+            },
+            {
+                "internalType": "uint8",
+                "name": "_numEpochKeyNoncePerEpoch",
+                "type": "uint8"
+            },
+            {
+                "internalType": "uint8",
+                "name": "_maxReputationBudget",
+                "type": "uint8"
+            },
+            {
+                "internalType": "uint256",
+                "name": "_epochLength",
+                "type": "uint256"
+            },
+            {
+                "internalType": "uint256",
+                "name": "_attestingFee",
+                "type": "uint256"
+            }
+        ],
+        "stateMutability": "nonpayable",
+        "type": "constructor"
+    },
+    {
+        "inputs": [
+            { "internalType": "address", "name": "attester", "type": "address" }
+        ],
+        "name": "AttesterAlreadySignUp",
+        "type": "error"
+    },
+    {
+        "inputs": [
+            {
+                "internalType": "uint256",
+                "name": "attesterId",
+                "type": "uint256"
+            }
+        ],
+        "name": "AttesterIdNotMatch",
+        "type": "error"
+    },
+    {
+        "inputs": [
+            { "internalType": "address", "name": "attester", "type": "address" }
+        ],
+        "name": "AttesterNotSignUp",
+        "type": "error"
+    },
+    { "inputs": [], "name": "AttestingFeeInvalid", "type": "error" },
+    { "inputs": [], "name": "EpochNotEndYet", "type": "error" },
+    { "inputs": [], "name": "EpochNotMatch", "type": "error" },
+    { "inputs": [], "name": "InvalidBlindedUserState", "type": "error" },
+    { "inputs": [], "name": "InvalidEpochKey", "type": "error" },
+    { "inputs": [], "name": "InvalidNumberBlindedHashChain", "type": "error" },
+    { "inputs": [], "name": "InvalidNumberNullifiers", "type": "error" },
+    { "inputs": [], "name": "InvalidProofIndex", "type": "error" },
+    {
+        "inputs": [
+            {
+                "internalType": "enum IUnirep.AttestationFieldError",
+                "name": "",
+                "type": "uint8"
+            }
+        ],
+        "name": "InvalidSNARKField",
+        "type": "error"
+    },
+    { "inputs": [], "name": "InvalidSignUpFlag", "type": "error" },
+    { "inputs": [], "name": "InvalidSignals", "type": "error" },
+    { "inputs": [], "name": "InvalidSignature", "type": "error" },
+    { "inputs": [], "name": "InvalidTransitionEpoch", "type": "error" },
+    {
+        "inputs": [
+            {
+                "internalType": "bytes32",
+                "name": "nullilier",
+                "type": "bytes32"
+            }
+        ],
+        "name": "NullilierAlreadyUsed",
```

Comment:
The typos in these functions need to be updated. 2851b2eca5ce28560ac1202bc949e769d2eeddad should have all the changed names.